app/src/main/java/net/aiscope/gdd_app/ui/capture/CaptureImageActivity.kt

```diff
@@ -37,6 +37,7 @@ class CaptureImageActivity : AppCompatActivity(), CaptureImageView, CaptureFlow
     private lateinit var zoomController: ZoomController
 
     private lateinit var binding: ActivityCaptureImageBinding
+    private lateinit var toast: Toast;
```

Comment:
No semicolons in Kotlin, please 😛 Unless it's for separating multiple statements on the same line.
```suggestion
    private lateinit var toast: Toast
```